modules/deploy/partials/linux/install-redpanda.adoc

```diff
@@ -31,6 +31,40 @@ curl -1sLf 'https://dl.redpanda.com/nzc4ZYQK3WRGd9sy/redpanda/cfg/setup/bash.rpm
 sudo -E bash && sudo yum install redpanda -y
 ----
 
+[IMPORTANT]
+====
+If you intend to install specific versions of Redpanda that are older than the latest available version, then you must specify the full list of versions
+for each package. For example, if installing `24.2.1~rc1-1`, and you fail to specify the version in the package, you may encounter the following error:
```

Comment:
```suggestion
To install a version of Redpanda that is older than the latest available version, you must also specify each dependency. For example, if installing `24.2.1~rc1-1`, and you fail to specify each package and its version, you may encounter the following error:
```

Reply:
I found it easier to process this way, but your choice. If you change this Fedora/RedHat section, please do the same for Debian/Ubuntu section below!